dataset/src/main/scala/frameless/functions/Udf.scala

```diff
@@ -0,0 +1,80 @@
+package frameless.functions
```

Comment:
good catch! by the way, I was always wondering what the difference of the two approaches is ... ?